### ISSUE TITLE
"Rollup with rollup-stream" (bad) recipe

### ISSUE THREAD

**Issue by yairEO** [gulpjs/gulp#2439]
The recipe isn't so good. 

Example shows:

```javascript
gulp.task('rollup', function() {
  return rollup({
      entry: './src/main.js'
    })

    // give the file the name you want to output with
    .pipe(source('app.js'))

    // and output to ./dist/app.js as normal.
    .pipe(gulp.dest('./dist'));
})
```

But the named key `entry` should be `input` (in other examples on the web) and also I get an error:

    Error: You must specify options.format, which can be one of 'amd', 'cjs', 'es', 'iife' or 'umd'

which isn't mentioned anywhere in the recipe. 

After trying to configure the format, the error sadly remains:

```
{
  input:entry,
  output: {
    file: outputName,
    format: 'cjs'
  }
}
```

**Comment by yairEO** [comment]
This was helpful: https://github.com/rollup/rollup/issues/1573#issuecomment-324539480

**Comment by contra** [comment]
@yairEO Can you send a PR to update the recipe example if it doesn't work anymore? Thanks!

**Comment by phated** [comment]
I'm just planning to delete this. All recipes are "bad" and haven't been made "good" following gulp 4 practices.

**Comment by wessuxx** [comment]
go a head dude

**Comment by phated** [comment]
This was actually fixed by #2466 and I didn't realize it. Closing this now.
